### PR TITLE
feat: add Swipe Review setting

### PR DESCRIPTION
**Pull request by MichaelKohler**
This adds the setting to enable the "Swiping Review" that @Adrijaned implemented in https://github.com/Common-Voice/sentence-collector/pull/264.

@freaktechnik if you want to review :)
@Adrijaned: if you want to test to make sure it works as desired

### CODE REVIEW

File: shared/db/collections/user.js [hunk 1]
```diff
@@ -70,6 +70,20 @@ export default class User {
...
+    const record = await collection.getRecord(this.username);
+    const user = record.data;
+
+    if (!user.settings) {
+      user.settings = {};
+    }
+
+    user.settings[key] = value;
+    const updatedUser = await collection.updateRecord(user);
+    return updatedUser.data.languages;
```
Comment (freaktechnik) [inline shared/db/collections/user.js]: That's an interesting thing to return

File: shared/db/collections/user.js [hunk 2]
```diff
@@ -70,6 +70,20 @@ export default class User {
...
+  async setSetting(key, value) {
+    const collection = await this.getCollection();
+    const record = await collection.getRecord(this.username);
+    const user = record.data;
+
+    if (!user.settings) {
+      user.settings = {};
+    }
+
+    user.settings[key] = value;
```
Comment (freaktechnik) [inline shared/db/collections/user.js]: Immutability is overrated

File: web/src/actions/index.js [hunk 3]
```diff
@@ -61,6 +63,22 @@ function checkPassword(password) {
...
+  return async function(dispatch, getState) {
+    try {
+      const state = getState();
+      const db = new WebDB(state.app.username, state.app.password);
+      await db.setSetting(key, value);
+      dispatch(settingsChanged({
+        [key]: value,
+      }));
+    } catch (err) {
+      dispatch(addLanguageFailure());
```
Comment (freaktechnik) [inline web/src/actions/index.js]: Settings are languages, suuure.

File: web/src/components/pages/profile.jsx [hunk 4]
```diff
@@ -162,12 +182,26 @@ export default class Profile extends React.Component {
...
-                            filters={this.props.languages} />
-          <button disabled={this.props.pending}
+                            filters={languages} />
+          <button disabled={pending}
                   onClick={this.onAdd} className="add-language">Add</button>
         </section>
+
+        <section>
+          <h2>Settings</h2>
+          <p>Experimental: There are two different tools with which you can review sentences. The normal tool lists 5 sentences per page
```
Comment (freaktechnik) [inline web/src/components/pages/profile.jsx]: Interesting wrapping.

File: web/src/components/pages/profile.jsx [hunk 5]
```diff
@@ -162,12 +182,26 @@ export default class Profile extends React.Component {
...
+          and has an approval and rejection button each. The Swiping tool displays one card at a time and you can swipe left
```
Comment (Adrijaned) [inline web/src/components/pages/profile.jsx]: Just my opinion:
```suggestion
          and has an approval and rejection button each. The Swiping tool displays one card at a time where you can swipe left
```

Comment (Adrijaned) [reply web/src/components/pages/profile.jsx]: Also, I would consider changing from swiping left __and__ right to left __or__ right (the same with approve/reject), and, changing either the order of `left and right` or `approve and reject` to match the order of actions with the order of decisions (current wording seems to imply to me you swipe left to approve and swipe right to reject)

File: web/src/components/pages/profile.jsx [hunk 6]
```diff
@@ -162,12 +182,26 @@ export default class Profile extends React.Component {
...
+          and right to approve and reject sentences. Both work on Desktop, for touch interfaces we would suggest to try out
```
Comment (Adrijaned) [inline web/src/components/pages/profile.jsx]:
```suggestion
          and right to approve and reject sentences. Both are well usable on desktop, but for touch interfaces we suggest that you try out
```

**Pull request merged**